jerry-core/ecma/builtin-objects/ecma-builtin-symbol-prototype.c

```diff
@@ -89,6 +89,47 @@ ecma_builtin_symbol_prototype_object_to_primitive (ecma_value_t this_arg) /**< t
   return ecma_builtin_symbol_prototype_object_value_of (this_arg);
 } /* ecma_builtin_symbol_prototype_object_to_primitive */
 
+/**
+ * The Symbol.prototype object's 'description' routine
+ *
+ * See also:
+ *          ECMA-262 v11, 19.4.3.2
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+ecma_builtin_symbol_prototype_description_getter (ecma_value_t this_arg) /**< this argument */
+{
+  if (ecma_is_value_symbol (this_arg))
```

Comment:
There is already a `ecma_symbol_to_string_helper` function which almost do the same. The second parameter of this function currently is boolean to decide whether `toString` or `valueOf` routine should be performed. Please change this boolean into an enum, and add `description` routine as well into this list. After that please modify this helper function according to it.